index.js

```diff
@@ -1,4 +1,91 @@
-//cat's are always hungry
-var now = new Date();
-// the cat says
-console.log('Feed the kitties at ' + now);
+window.oranjs = window.oranjs || {};
+
+oranjs.Globals = {
```

Comment:
Checkout my repo `fireball`. It's a Tamagotchi app that I used to teach JavaScript to teenagers and I have some issues in there as well.